src/ray/gcs/store_client/in_memory_store_client.h

```diff
@@ -50,14 +51,22 @@ class InMemoryStoreClient : public StoreClient {
 
   Status AsyncDelete(const std::string &table_name,
                      const std::string &key,
-                     const StatusCallback &callback) override;
+                     std::function<void(bool)> callback) override;
 
   Status AsyncBatchDelete(const std::string &table_name,
                           const std::vector<std::string> &keys,
-                          const StatusCallback &callback) override;
+                          std::function<void(int64_t)> callback) override;
 
   int GetNextJobID() override;
```

Comment:
This function is wired to be put here. Should we move it outside?

Reply:
yeah it should be a counter instead. we will fix it in following PRs.